validator/src/main.rs

```diff
@@ -1057,9 +1070,10 @@ pub fn main() {
                 .value_name("SHRED_STORAGE_SIZE_BYTES")
                 .takes_value(true)
                 .validator(is_parsable::<u64>)
-                .default_value(default_rocksdb_fifo_shred_storage_size)
                 .help("The shred storage size in bytes. \
-                       The suggested value is 50% of your ledger storage size in bytes."),
+                       The suggested value is at least 50% of your ledger storage size. \
+                       If this argument is unspecified, we will assign a proper \
+                       value based on --limit-ledger-size."),
```

Comment:
Now that we discovered we had previously not been honoring the case where `--limit-ledger-size` was unspecified (to allow unbounded growth), should we tweak the help a little to explain what we'll do in that case? I don't love my wording here, but to illustrate the point I'm making:

> If this argument is unspecified, we will choose a proper value consistent with --limit-ledger-size value, or allow unbounded storage size growth if unspecified

Reply:
> Now that we discovered we had previously not been honoring the case where --limit-ledger-size was unspecified (to allow unbounded growth), should we tweak the help a little to explain what we'll do in that case?

Sure thing.  Btw, to clarify a little bit.  So we have three cases:

* `--limit-ledger-size` is presented and a value is specified, in that case we will use the specified value.
* `--limit-ledger-size` is presented but no value is specified, in this case `DEFAULT_MAX_LEDGER_SHREDS` will be used, which is 200_000_000.
* `--limit-ledger-size` is not presented.  In this case, the ledger size is unlimited by definition and thus is unbounded.